src/datastore/StringCrypto.ts

```diff
@@ -48,19 +53,59 @@ export class StringCrypto {
     }
 
     public encrypt(plaintext: string): string {
-        const salt = crypto.randomBytes(16).toString('base64');
-        return crypto.publicEncrypt(
+        const salt = randomBytes(16).toString('base64');
```

Comment:
It would help readability / maintainability to define variables for the salt length & related values.

Example:

```ts
const SALT_ENCODING = 'base64';
const SALT_BYTE_LENGTH = 16;
const SALT_STRING_LENGTH = Buffer.alloc(SALT_BYTE_LENGTH).toString(SALT_ENCODING).length;

const ENCRYPTED_ENCODING = 'base64';

export class StringCrypto {
    // snip
    public encrypt(plaintext: string): string {
        const salt = randomBytes(SALT_BYTE_LENGTH).toString(SALT_ENCODING);
        return publicEncrypt(
            this.privateKey,
            Buffer.from(salt + ' ' + plaintext)
        ).toString(ENCRYPTED_ENCODING);
    }
    public decrypt(encryptedString: string): string {
        const decryptedPass = privateDecrypt(
            this.privateKey,
            Buffer.from(encryptedString, ENCRYPTED_ENCODING)
        ).toString();
        // Extract the password by removing the prefixed salt and seperating space
        return decryptedPass.slice(SALT_STRING_LENGTH+1);
    }
    // snip
}
```